numba_cuda/numba/cuda/cudadrv/nvvm.py

```diff
@@ -431,6 +428,21 @@ def get_log(self):
     ],
 }
 
+# From CUDA 12.9 Release notes, Section 1.5.4, "Deprecated Architectures"
+# https://docs.nvidia.com/cuda/archive/12.9.0/cuda-toolkit-release-notes/index.html#deprecated-architectures
+#
+#   "Maxwell, Pascal, and Volta architectures are now feature-complete with no
+#   further enhancements planned. While CUDA Toolkit 12.x series will continue
+#   to support building applications for these architectures, offline
+#   compilation and library support will be removed in the next major CUDA
+#   Toolkit version release. Users should plan migration to newer
+#   architectures, as future toolkits will be unable to target Maxwell, Pascal,
+#   and Volta GPUs."
+#
+# In order to maintain compatibility with future toolkits, we use Turing (7.5)
+# as the default CC if it is not otherwise specified.
+LOWEST_CURRENT_CC = (7, 5)
```

Comment:
It would be great if we could query this information somehow in the future instead of having to hardcode it. Probably gets easier once we depend on `cuda.bindings`?

Reply:
If NVRTC is a hard dependency, perhaps try the first arch reported by `nvrtc.nvrtcGetSupportedArchs()`?

Reply:
This is good to go so I'm going to merge this now and open this as a follow up issue.